### PR TITLE
Don't format generated file

### PR DESCRIPTION
**Pull request by zasdfgbnm**
Besides, fix https://github.com/NVIDIA/Fuser/issues/141

### CODE REVIEW

File: csrc/serde/fusion_cache_generated.h
Comment (zasdfgbnm) [inline csrc/serde/fusion_cache_generated.h:1]: I changed this file back to its originally generated version. It is very annoying that git keeps telling me that I changed this file locally.

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -86,38 +86,3 @@ jobs:
 
           # Run lintrunner except clang-tidy
           lintrunner --force-color --take FLAKE8,MYPY,CLANGFORMAT,NOQA,TYPEIGNORE,NEWLINE,MYPYSTRICT,TABS,SPACES,EXEC,BLACK --all-files
-
```
Comment (zasdfgbnm) [inline .github/workflows/lint.yml:89]: flake8 and clang-format is already covered by lintrunner, so it makes no sense to run it again. Begging for admin power to lift branch protection requirement.

**Review by rdspring1 — COMMENTED** [review]
LGTM. Formatting the generated file was getting annoying.

**Review by jjsjann123 — APPROVED** [review]

**Comment by jjsjann123** [comment]
Protection rule updated. @zasdfgbnm you may now proceed to merge.

image

**Pull request merged**